src/actix/mod.rs

```diff
@@ -64,11 +64,13 @@ pub fn init(
         let http_client = web::Data::new(HttpClient::from_settings(&settings)?);
         let health_checker = web::Data::new(health_checker);
         let auth_keys = AuthKeys::try_create(&settings.service);
-        let static_folder = settings
-            .service
-            .static_content_dir
-            .clone()
-            .unwrap_or(DEFAULT_STATIC_DIR.to_string());
+        let static_folder = env::var("STATIC_DIR").unwrap_or_else(|_| {
```

Comment:
If you want to overwrite value from settings in Qdrant, you can use the following pattern:

`QDRANT__SERVICE__STATIC_CONTENT_DIR=...` instead of introducing a new env variable that doesn't follow the pattern. More info here: https://qdrant.tech/documentation/guides/configuration/#environment-variables

Reply:
The only objection I have against this, is that this prevents the user from customizing the directory. But I guess that's an implementation detail we can resolve later.

Reply:
Sweet! Thank you for picking this up so quickly.